packages/shorebird_cli/lib/src/commands/account/subscribe_account_command.dart

```diff
@@ -0,0 +1,82 @@
+import 'dart:async';
+
+import 'package:mason_logger/mason_logger.dart';
+import 'package:shorebird_cli/src/command.dart';
+import 'package:shorebird_cli/src/shorebird_config_mixin.dart';
+import 'package:shorebird_code_push_client/shorebird_code_push_client.dart';
+
+/// {@template subscribe_account_command}
+/// `shorebird account subscribe`
+/// {@endtemplate}
+class SubscribeAccountCommand extends ShorebirdCommand
+    with ShorebirdConfigMixin {
+  /// {@macro subscribe_account_command}
+  SubscribeAccountCommand({
+    required super.logger,
+    super.auth,
+    super.buildCodePushClient,
+  });
+
+  @override
+  String get name => 'subscribe';
+
+  @override
+  String get description => 'Sign up for a Shorebird subscription.';
```

Comment:
How and where do they learn more about what subscription options are available?  (I don't have anything written, up but happy to.). 

https://github.com/shorebirdtech/shorebird/edit/main/TRUSTED_TESTERS.md is probably where we should point folks for now.  The Stripe link should also explain the product.

Reply:
Added a more detailed summary to the command. lmk what you think.

<img width="1081" alt="Screenshot 2023-04-21 at 12 29 00 PM" src="https://user-images.githubusercontent.com/581764/233687799-0153ae5c-c47b-4e05-a309-097b568a8152.png">

Reply:
lgtm.  I would just say "currently" $20 a month.

Reply:
Done.